### DISCUSSION TITLE
WM_MOUSEWHEEL It only works in one window.

### DISCUSSION THREAD

**Discussion by KxFxN** [mhammond/pywin32#2532] [General]
Is there any python that works like pywin32 SendMessage PostMessage. doesn't move the mouse?

I want to move the mouse scroll or click the mouse without moving the mouse, I have used WM_MOUSEWHEEL. but not successful. It only works in one window. Will there be another one

**or it only works on the left side of the screen I tried moving the window to another section but it didn't work when I moved it back to the same place in the top left it works.**

this is an example:
https://drive.google.com/file/d/137ouHgJR3eRZTUOFkHjPGO2eoMfQelCz/view?usp=sharing

**Comment by mhammond** [comment]
https://github.com/mhammond/pywin32/blob/master/README.md#support

**Comment by KxFxN** [comment]
> https://github.com/mhammond/pywin32/blob/master/README.md#support

what do you mean?

I do not understand

**Comment by mhammond** [comment]
That link says "However, please do not open github issues for general support requests, or for problems or questions using the modules in this package - they will be closed. For such issues, please email the [python-win32 mailing list](http://mail.python.org/mailman/listinfo/python-win32) - note that you must be subscribed to the list before posting." - your issue is very clearly a question about how to use these modules, so you need to follow those instructions.

**Comment by KxFxN** [comment]
> ลิงก์นั้นระบุว่า "อย่างไรก็ตามโปรดอย่าเปิดปัญหา github สําหรับคําขอรับการสนับสนุนทั่วไปหรือสําหรับปัญหาหรือคําถามที่ใช้โมดูลในแพ็คเกจนี้ - พวกเขาจะถูกปิด สําหรับปัญหาดังกล่าวโปรดส่งอีเมล[รายชื่อผู้รับจดหมาย python-win32](http://mail.python.org/mailman/listinfo/python-win32) - โปรดทราบว่าคุณต้องสมัครสมาชิกรายการก่อนโพสต์" - ปัญหาของคุณเป็นคําถามที่ชัดเจนมากเกี่ยวกับวิธีการใช้โมดูลเหล่านี้ดังนั้นคุณต้องทําตามคําแนะนําเหล่านั้น

what should i do I would like some advice on how to use it. or problem solving

**Comment by mhammond** [comment]
You should follow those instructions - subscribe to that mailing list and post your question there.
